koopovereenkomst-v3-simple/src/koek/KoekCommandHandler.ts

```diff
@@ -0,0 +1,129 @@
+import { default as solidQuery } from "@solid/query-ldflex/lib/exports/rdflib";
+import { v4 as uuidv4 } from "uuid";
+import { Event } from "../../src/koek/Event";
+import KoekAggregate from "./KoekAggregate";
+import KoekRepository from "./KoekRepository";
+
+
+export default class KoekCommandHandler {
```

Comment:
Standaard patroon in de Command Handler wordt:

1. `koek.pushEvent()`
2. `koek.processEvents()`

Ik denk dat dat wel mooi is en lijkt op de Event Sourcing patronen in (bijv.) Axon Framework ... zonder de overhead van een heel framework (wat waarschijnlijk veel rijker aan variaties zal zijn, maar dat hebben we nu (nog) niet echt nodig ... 😉 )